src/Domain_Item.php

```diff
@@ -546,7 +549,7 @@ public static function showForItem(CommonDBTM $item, $withtemplate = 0)
                 $relation_names[$data['domainrelations_id']] = Dropdown::getDropdownName(table: "glpi_domainrelations", id: $data['domainrelations_id'], default: '');
             }
 
-            $expiration = $data["date_expiration"];
+            $expiration = htmlspecialchars(Html::convDate($data["date_expiration"]));
```

Comment:
Shouldn't this just be set to `$data['date_expiration']` and then set the formatter to 'date'?